assets/wizards/site-design/views/main/index.js

```diff
@@ -118,32 +177,46 @@ const Main = ( {
 				title={ __( 'Typography', 'newspack' ) }
 				description={ __( 'Pick the font pairing to use throughout your site', 'newspack' ) }
 			/>
+			<RadioControl
```

Comment:
This may not be related to these changes, but it's the first time I'm noticing. The "selected" radio button style looks off to me (using WP 5.7.2 without the Gutenberg plugin, tested with Chrome and Firefox and replicated in local dev site and on a JN site):

<img width="128" alt="Screen Shot 2021-06-09 at 4 36 49 PM" src="https://user-images.githubusercontent.com/2230142/121438497-528d9600-c941-11eb-96e2-51f575508ef8.png">

Reply:
I might also suggest a slightly different UI: instead of using two radio buttons, why not a single `ToggleControl`, e.g.

<img width="167" alt="Screen Shot 2021-06-09 at 4 43 00 PM" src="https://user-images.githubusercontent.com/2230142/121438729-c891fd00-c941-11eb-8fee-13ef9bdcf097.png">

Reply:
Also, if switching from custom to default/curated, it might be worth noting that your custom import code or URL will be lost after saving.

Reply:
> The "selected" radio button style looks off to me 

Yes - I forgot to mention that. Could you take a look @thomasguillot ?

> I might also suggest a slightly different UI: instead of using two radio buttons, why not a single ToggleControl, e.g.

I second that - WDYT Thomas?

Reply:
> The "selected" radio button style looks off to me

Hmm yeah weird... fixed https://github.com/Automattic/newspack-plugin/pull/999/commits/3e7f0d3cd22336a04e145d74a40d3de6c6ec4e3c

Reply:
> I might also suggest a slightly different UI: instead of using two radio buttons, why not a single ToggleControl, e.g.

I used radio buttons on purpose as it felt more balanced visually. I also wanted to limit the user’s choice to one option whilst displaying both options: you can choose our curated list OR import your own custom fonts.

![Screenshot 2021-06-10 at 15 37 03](https://user-images.githubusercontent.com/177929/121544933-fb9bc580-ca01-11eb-9c8a-4a5829441470.png)
![Screenshot 2021-06-10 at 15 38 02](https://user-images.githubusercontent.com/177929/121544936-fc345c00-ca01-11eb-8b0d-bb80e83a1504.png)

Reply:
>  if switching from custom to default/curated, it might be worth noting that your custom import code or URL will be lost after saving

I wouldn't worry too much about this. This setting tends to be set once, and forget about it. And, if a publication has its own custom fonts, it's unlikely they will play with the curated list.

Reply:
> I used radio buttons on purpose as it felt more balanced visually. I also wanted to limit the user’s choice to one option whilst displaying both options: you can choose our curated list OR import your own custom fonts.

Ok, I'll defer to you on this. One thing, though, do you think we can change the label "Curated list" to "Default fonts" instead? I think that would be more descriptive and it mirrors "Custom fonts" a bit more.

Reply:
> Hmm yeah weird... fixed 3e7f0d3

Confirmed, thanks!

Reply:
> Ok, I'll defer to you on this. One thing, though, do you think we can change the label "Curated list" to "Default fonts" instead? I think that would be more descriptive and it mirrors "Custom fonts" a bit more. (I also worry that the term "curated list" might be confused with the same term as used in Newspack Listings, for a completely different purpose!)

Done in https://github.com/Automattic/newspack-plugin/pull/999/commits/da426b542be4f3571d08ab50725fffa9c4458933

I removed "fonts" from the label since it felt redundant

Reply:
Looks better, thanks!